student.rb

```diff
@@ -3,7 +3,7 @@ class Student < Person
   attr_accessor :classroom
 
   def initialize(age:, classroom:, name: 'Unknown', parent_permission: true)
-    super(age: age, name: name, parent_permission: parent_permission)
+    super(name: name, age: age, parent_permission: parent_permission)
     @classroom = classroom
   end
```

Comment:
- [ ] This class is expected to have a classroom method that serves as the 'belongs-to' side (a student belongs to a classroom). Kindly add that 👍🏽